CityOfDetroit/bloom
# UI for editing/adding units summaries

## Issue

- Closes #278 

## Description

Creates a modal to edit the units summary table. The style is consistent with the units editing table.

Specific changes made:
- Added a new form for adding / editing a units summary. It will look a lot like the add/edit unit form, because I copied it from there.
- Removed `listing` (type `IdDto`) from `UnitsSummaryDto`. It was never referenced and isn't necessary.
- Added translations for language in the new form.

Example of table I populated with data via the UI:
![image](https://user-images.githubusercontent.com/83078310/131190970-978808a2-70b5-434b-a3df-3141d495c20e.png)

The summary edit modal:
![image](https://user-images.githubusercontent.com/83078310/131175716-21ad82fa-3abf-4a1e-8e26-b1cc8300e113.png)

Example of a listing that I added and populated with a summary:
![image](https://user-images.githubusercontent.com/83078310/131175757-702cb084-9b43-41b1-ad82-38cca220f7da.png)


## Type of change


## How Can This Be Tested/Reviewed?

Workflows I tested:
- Adding a summary to an existing listing
- Editing an existing summary
- Adding / editing a summary in a new listing
- Deleting a summary

NOTE: If you try to play around with this, you may encounter an error where the listing won't update. This can happen if you try to update a listing that doesn't have a leasing agent email. Add a test email to the form and try to submit again and it should work.


## Checklist:
